modules/core/src/passes/shadow-pass.js

```diff
@@ -1,52 +1,43 @@
 import {default as LayersPass} from './layers-pass';
-import {
-  Framebuffer,
-  Texture2D,
-  Renderbuffer,
-  withParameters,
-  cssToDeviceRatio
-} from '@luma.gl/core';
+import {isWebGL2, Framebuffer, Texture2D, withParameters, cssToDeviceRatio} from '@luma.gl/core';
+import GL from '@luma.gl/constants';
 
 export default class ShadowPass extends LayersPass {
   constructor(gl, props) {
     super(gl, props);
 
-    // The shadowMap texture
-    this.shadowMap = new Texture2D(gl, {
-      width: 1,
-      height: 1,
-      parameters: {
-        [gl.TEXTURE_MIN_FILTER]: gl.LINEAR,
-        [gl.TEXTURE_MAG_FILTER]: gl.LINEAR,
-        [gl.TEXTURE_WRAP_S]: gl.CLAMP_TO_EDGE,
-        [gl.TEXTURE_WRAP_T]: gl.CLAMP_TO_EDGE
-      }
-    });
-
-    this.depthBuffer = new Renderbuffer(gl, {
-      format: gl.DEPTH_COMPONENT16,
-      width: 1,
-      height: 1
-    });
+    if (Framebuffer.isSupported(gl, {colorBufferFloat: true})) {
+      // The shadowMap texture
+      this.shadowMap = new Texture2D(gl, {
+        format: isWebGL2(gl) ? GL.RGBA32F : GL.RGBA,
+        type: GL.FLOAT,
+        width: 1,
+        height: 1,
+        parameters: {
+          [GL.TEXTURE_MIN_FILTER]: GL.LINEAR,
+          [GL.TEXTURE_MAG_FILTER]: GL.LINEAR,
+          [GL.TEXTURE_WRAP_S]: GL.CLAMP_TO_EDGE,
+          [GL.TEXTURE_WRAP_T]: GL.CLAMP_TO_EDGE
+        }
+      });
 
-    this.fbo = new Framebuffer(gl, {
-      id: 'shadowmap',
-      width: 1,
-      height: 1,
-      attachments: {
-        [gl.COLOR_ATTACHMENT0]: this.shadowMap,
-        // Depth attachment has to be specified for depth test to work
-        [gl.DEPTH_ATTACHMENT]: this.depthBuffer
-      }
-    });
+      this.fbo = new Framebuffer(gl, {id: 'shadowmap'});
```

Comment:
Nit: remove fbo.attach call by passing `attachments: {[GL.CO...} object to constructor.

Reply:
If I supply `attachments` then no depth attachment will be automatically created.